swaylock/seat.c

```diff
@@ -88,6 +88,7 @@ static void keyboard_modifiers(void *data, struct wl_keyboard *wl_keyboard,
 	xkb_mod_mask_t mask = xkb_state_serialize_mods(state->xkb.state,
 			XKB_STATE_MODS_DEPRESSED | XKB_STATE_MODS_LATCHED);
 	state->xkb.modifiers = 0;
+	state->xkb.caps_lock = mods_locked & 0x02;
```

Comment:
Isn't there a constant for this one?

Reply:
There is actually WLR_MODIFIER_CAPS in wlr/types/wlr_keyboard.h. However, including that caused some errors due to deprecation... And besides that one, I didn't find an obvious one...